packages/core/nuxt-module/lib/module.js

```diff
@@ -45,36 +45,42 @@ module.exports = function VueStorefrontNuxtModule (moduleOptions) {
 
   log.info(chalk.green('Starting Vue Storefront Nuxt Module'))
 
+  // Add meta data
   this.options.head.meta.push({
     name: 'generator',
     content: 'Vue Storefront 2'
-  })
+  });
+
+  // Enable HTTP/2 push for JS files
+  if (options.performance.httpPush) {
+    this.options.render = merge(this.options.render, {
+      http2: {
+        push: true,
+        pushAssets: (request, response, publicPath, preloadFiles) => {
+          return preloadFiles
+            .filter(({ asType }) => asType === 'script')
+            .map(({ file, asType }) => `<${publicPath}${file}>; rel=preload; as=${asType}`);
+        }
+      }
+    });
+  }
   
+  // SSR plugin
   this.addPlugin(path.resolve(__dirname, 'plugins/ssr.js'))
   log.success('Installed Vue Storefront SSR plugin');
 
+  // Logger plugin
   this.addPlugin({
     src: path.resolve(__dirname, 'plugins/logger.js'),
     options: moduleOptions.logger || {}
   })
   log.success('Installed VSF Logger plugin');
 
+  // Composition API plugin
   this.addModule('@nuxtjs/composition-api')
   log.success('Installed nuxt Composition API Module');
 
-  //-------------------------------------
-
-  // Using symlinks in lerna somehow breaks composition API behavior as a singleton.
```

Comment:
This is no longer needed, because now every package should use shared `@vue/composition-api` from root of the projects (related to `nohoist` task).